pkgs/development/tools/gotools/default.nix

```diff
@@ -2,13 +2,12 @@
 
 buildGoModule rec {
   pname = "gotools-unstable";
```

Comment:
Is it `unstable` if we're using a tag?
```suggestion
  pname = "gotools";
```

Reply:
Good catch, thanks. Fixed in the same push that added the meta section.